duoStrength.js

```diff
@@ -513,10 +513,11 @@ function addStrengths(strengths)
 			strengthBarHolder.className = "strengthBarHolder";
 			strengthBarHolder.style = 
 			`
-				width: 120%;
+				width: 100%;
 				height: 1.4em;
 				position: relative;
-				left: -10%;
+				// left: -10%;

```

Comment:
Should probably just be removed, a comment in a string template doesn't really work, see [here](https://stackoverflow.com/questions/40046764/is-it-possible-to-have-a-comment-inside-a-es6-template-string)
```suggestion
```